openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGTrainData.java

```diff
@@ -90,17 +105,30 @@ public class SWIGTrainData implements AutoCloseable {
      * After that the BoosterHandle and the DatasetHandle will still need to be initialized at the proper times:
      * @see SWIGTrainData#initSwigDatasetHandle()
      *
+     * TODO: overload this constructor with a default argument fairnessConstrained=false
```

Comment:
Remove this comment and fix the docstrings of both overloads, which are outdated/non-existent.